inventory/sample.aws.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -9,10 +9,23 @@ ansible_become: true
 # Node Labels to use with the CASL tools
 openshift_node_labels: "{{ ec2_tag_node_labels }}"
 
+# CNS relative vars
+openshift_storage_glusterfs_namespace: glusterfs
+openshift_storage_glusterfs_name: cns
+
+# Uncommented to automatically integrate OCP with AWS so EBS can be used as Dynamic Storage backend
```

Comment:
These should be enabled by default, especially since `cnd_deploy` is set to `false` below

Reply:
Done